### DISCUSSION TITLE
project origins: drama question

### DISCUSSION THREAD

**Discussion by cdnninja** [dahlb/ha_kia_hyundai#41] [Q&A] [answered]
Thought I would post here since the other thread you post is locked.  

You specifically tagged me more than once mentioning fixes for me and offered that I backport.  I see now you are saying I was in the wrong for doing so. I also provided credit in the key one in the PR (multi car). 

Are you requesting I don't back port when you tag me? 

I want to support the community but not looking to create drama.   Making me not want to contribute to either project to be honest.

**Comment by dahlb** [comment, marked as answer]
I am sorry to have involved you in the toxic drama that has unfolded between me and that other fork's owner, I will try to improve my post about the origin of this fork to clarify that wasn't my intention.

I bothered to make my code open to allow for porting and don't want you to feel uncomfortable doing so, it's MIT licensed after all.

I am just upset that I was banned from the other repo, so now I can't even watch for community activity and ensure my fork benefits from the general community feedback.

I feel I contributed plenty directly to that origin fork as I did pretty much the entire us kia implementation and the asynchronous api result tracking for USA kia which got ported to the ca implementation very well by you.

It took me sadly nearly two weeks to find that no force scan bug, that was the main problem I had in the original fork, but couldn't track down until I did a full rewrite in my green field branch.  I think the ban combined with my previous contributions there both documented in git and ported was very upsetting, and I agree my other post focused on the wrong things and I will try to fix it; but I think that's the origin of that.  

I hope that makes sense, and I'll try to improve the other post now. Which I made to try and clarify my side after being banned and maligned in my last post on the other fork.

**Reply by dahlb** [reply]
I updated my other post, sorry again for involving you in the first version.  You have been great, and I have really appreciated your workflows on this fork. Your enthusiasm for finding solutions and moving the community forward had a lot to do with my getting the will to start programming again through this project after a year of no programming and never before using python.  I hope you can stay clear of this drama and other such drama in open source projects and continue contributing, whether to either of these forks or another project entirely.
